search/filters/FulltextFilter.php

```diff
@@ -73,5 +73,18 @@ public function getDbName() {
 
 		return parent::getDbName();
 	}
+	
+	private function prepareColumns($columns) {
+		$class = '';
+		$parts = explode(',', $columns);
+		array_walk($parts, function(&$col, $key) use (&$class) {
+			$name = trim($col, " \t\n\r\0\x0B\"");
```

Comment:
I wonder if there is a consistently used constant or some reusable resource that contains all possible characters to trim here, rather than having them specified as a string.